Adyen/UI/View Controllers/AddressLookup/Search/AddressLookupSearchViewController+Style.swift

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+public struct AddressLookupSearchStyle: ViewStyle {
```

Comment:
public type missing documentation

Reply:
Fixed in: 1f2519998a83402396d39b279bf7e24a717a8213